guides/release/using-typescript/application-development/index.md

```diff
@@ -0,0 +1,3 @@
+---
+redirect: using-typescript/application-development/configuration
```

Comment:
same here, why do we have this redirect?

Reply:
IIRC clicking the top-level header was broken w/o an "index.md". Will double-check.

Reply:
ah yes that will probably be true, but there should be an index and it should have content instead of a redirect ;) in this case you would just need to rename the `configuration` file to be `index.md` and then that would be the first thing in the section 👍 the name of the page can still be `Configuration` it just changes the url structure

Reply:
I have some ideas to fix. Please hold.